src/action_mapper.ml

```diff
@@ -0,0 +1,47 @@
+open! Stdune
+
+module Make
+    (Src : Action_intf.Ast)
+    (Dst : Action_intf.Ast)
+= struct
```

Comment:
Can we include an .mli for this module? I assume you had to split it because of circular deps?

Reply:
Yes, it was to break a dependency cycle. I refrained from adding an `mli` because both functions need to be exposed and they have a long type that needs to be repeated, and it is not convenient to add a type component to the signature in order to share this type because the module is `included`-ed in other modules that may define similarly-named types.